db/migrate/20200627080640_create_conversations.rb

```diff
@@ -0,0 +1,8 @@
+class CreateConversations < ActiveRecord::Migration[5.2]
+  def change
+    create_table :conversations do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
What are you using this model for? The messaging part is already working, as you have tested in that specific feature. You shouldn't have to generate anything new to make this feature work. 
Stay in scope 😉